doc/source/pmm-admin.rst

```diff
@@ -171,30 +178,30 @@ monitoring service parameters along with the :program:`pmm-admin add` command
 when adding the :option:`mongodb:queries` monitoring service.
 
 .. code-block:: bash
-   :caption: Passing an SSL/TLS parameter to :program:`mongod` to enables 
+   :caption: Passing an SSL/TLS parameter to :program:`mongod` to enables
              a TLS connection.
 
-   $ sudo pmm-adming add mongodb:queries -- mongodb.tls
+   $ sudo pmm-adming add mongodb:queries -- --mongodb.tls
 
 .. list-table:: Supported SSL/TLS Parameters
    :widths: 25 75
    :header-rows: 1
 
    * - Parameter
      - Description
-   * - :option:`-mongodb.tls`
+   * - :option:`--mongodb.tls`
```

Comment:
I'm not sure about this cosmetic change. `--help` returns flags in this form:
```
/usr/local/percona/pmm-client/mongodb_exporter 0.3.1 exports various MongoDB metrics in Prometheus format.
Usage: /usr/local/percona/pmm-client/mongodb_exporter [flags]

Flags:
  -groups.enabled string
    	Comma-separated list of groups to use, for more info see: docs.mongodb.org/manual/reference/command/serverStatus/ (default "asserts,durability,background_flushing,connections,extra_info,global_lock,index_counters,network,op_counters,op_counters_repl,memory,locks,metrics")
  -log.format value
    	Set the log target and format. Example: "logger:syslog?appname=bob&local=7" or "logger:stdout?json=true" (default "logger:stderr")
  -log.level value
    	Only log messages with the given severity or above. Valid levels: [debug, info, warn, error, fatal]
  -mongodb.tls
    	Enable tls connection with mongo server
  -mongodb.tls-ca string
    	Path to PEM file that contains the CAs that are trusted for server connections.
    	If provided: MongoDB servers connecting to should present a certificate signed by one of this CAs.
    	If not provided: System default CAs are used.
  -mongodb.tls-cert string
    	Path to PEM file that contains the certificate (and optionally also the private key in PEM format).
    	This should include the whole certificate chain.
    	If provided: The connection will be opened via TLS to the MongoDB server.
  -mongodb.tls-disable-hostname-validation
    	Do hostname validation for server connection.
  -mongodb.tls-private-key string
    	Path to PEM file that contains the private key (if not contained in mongodb.tls-cert file).
  -mongodb.uri string
    	MongoDB URI, format: [mongodb://][user:pass@]host1[:port1][,host2[:port2],...][/database][?options] (default "mongodb://localhost:27017")
  -version
    	Print version information and exit.
  -web.auth-file string
    	Path to YAML file with server_user, server_password options for http basic auth (overrides HTTP_AUTH env var).
  -web.listen-address string
    	Address to listen on for web interface and telemetry. (default ":9216")
  -web.metrics-path string
    	Path under which to expose metrics. (default "/metrics")
  -web.ssl-cert-file string
    	Path to SSL certificate file.
  -web.ssl-key-file string
    	Path to SSL key file.
```
So I think it's ok we stay with `-` in docs for mongodb_exporter to avoid confusing users.